201-logic-app-as2-send-receive/README.md

```diff
@@ -13,35 +13,59 @@
 [![Deploy To Azure US Gov](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazuregov.svg?sanitize=true)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-logic-app-as2-send-receive%2Fazuredeploy.json)  
 [![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.svg?sanitize=true)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-logic-app-as2-send-receive%2Fazuredeploy.json)
 
-This template demonstrates AS2 Send Receive using Logic Apps. It creates Integration Accounts for two partners (Contoso and Fabrikam) and adds Partners and Agreements into them. It creates Logic Apps between Fabrikam Sales and Contoso which demonstrate Sync AS2 Send Receive. It also creates Logic Apps between Fabrikam Finance and Contoso which demonstrate ASync AS2 Send Receive.
+This template creates an AS2 Send Receive workflow by using Azure Logic Apps to create the following Azure resources:
+
+* Integration accounts for two partners, Contoso and Fabrikam, along with the necessary artifacts, partners and agreements.
+* Logic apps between Fabrikam Sales and Contoso, which demonstrate Sync AS2 Send Receive.
+* Logic apps between Fabrikam Finance and Contoso, which demonstrate ASync AS2 Send Receive.
+
+> [!IMPORTANT]
```

Comment:
this is not valid markdown

Reply:
Replaced asterisks with dashes, thanks!

Reply:
It's not the bullets - it's the 

[!IMPORTANT]

syntax, not sure what you want that to do, but in github it appears literally.

Reply:
@bmoore-msft, thanks I fixed the formatting on those notes. In our doc repo, that note style is supported.